photonjs/README.md

```diff
@@ -161,6 +162,44 @@ type Profile = {
   - Terminating/chainable operation
   - Graph selection: Travesal vs operate
   - ...
+  
+## Debugging
+```ts
+
+// Either like this
+const photon = new Photon({
+  logQueries: 'pipe',
+})
+
+// Or like this
+const photon = new Photon({
+  debug: {
```

Comment:
Is it also possible to provide `debug: true`?

Reply:
Just moved it out of `debug`.